logic/transaction.js

```diff
@@ -407,15 +407,22 @@ Transaction.prototype.process = function (trs, sender, requester, cb) {
  * @param {transaction} trs
  * @param {account} sender
  * @param {account} requester
+ * @param {boolean} checkPersistent - Check if trs already persisted in database
  * @param {function} cb
  * @return {setImmediateCallback} validation errors | trs
  */
-Transaction.prototype.verify = function (trs, sender, requester, cb) {
+Transaction.prototype.verify = function (trs, sender, requester, checkPersistent, cb) {
 	var valid = false;
 	var err = null;
 
-	if (typeof requester === 'function') {
-		cb = requester;
+	// Set default value of param if not provided
+	if(checkPersistent === undefined || checkPersistent === null) {
```

Comment:
Default to `true` is not good, we should call that function with `checkPersistent` either `true` or `false`, so that will be not needed.